src/organization/screens/organization-profile.screen.js

```diff
@@ -205,6 +182,32 @@ class OrganizationProfile extends Component {
   }
 }
 
-export const OrganizationProfileScreen = connect(selectors, actions)(
-  OrganizationProfile
-);
+const mapStateToProps = (state, ownProps) => {
+  // TODO: This should be normalized to params.id
+  const orgId = ownProps.navigation.state.params.organization.login.toLowerCase();
+
```

Comment:
Once implemented everywhere, screens should take just an "id" instead of an object like "user" or "organization".

The id suffice to get the data from the redux store, and we will standardize & simplify navigation greatly